contrib/jenkins_tests/rpm.sh

```diff
@@ -22,6 +22,8 @@ ${WORKSPACE}/configure --prefix=${rpm_dir}/install $jenkins_test_custom_configur
 
 if [ -x /usr/bin/dpkg-buildpackage ]; then
     echo "Build on debian"
+    ## To avoid "dpkg-checkbuilddeps: error: Unmet build dependencies: debhelper (>= 7)" on Ubuntu24.x
+    . /etc/os-release && [ "$VERSION_ID" = "24.04" ] && apt install -y debhelper 
```

Comment:
why cant you run it always? what ubuntu flavor does not have this package?

Reply:
VMs should have this package pre-installed and should have sudo installed, containers should all run as root.
Please give me example where this is not working.

Reply:
The problem in this particular CI is we're running an additional axis in the matrix and on MV (r-aa-fatty22-006) this causes apt/dpkg operation to happen at the same time and to make conflicts:
@default/r-aa-fatty22-006/1:
`
[2024-08-26T21:46:55.504Z] + echo 'Build on debian'

[2024-08-26T21:46:55.504Z] Build on debian
[2024-08-26T21:46:55.504Z] ++ id -u
[2024-08-26T21:46:55.504Z] + '[' 6213 -ne 0 ']'
[2024-08-26T21:46:55.504Z] + sudo apt-get install -y debhelper
[2024-08-26T21:46:55.504Z] Reading package lists...
[2024-08-26T21:46:55.504Z] Building dependency tree...
[2024-08-26T21:46:55.504Z] Reading state information...
[2024-08-26T21:46:55.504Z] debhelper is already the newest version (9.20160115ubuntu3).
[2024-08-26T21:46:55.504Z] 0 upgraded, 0 newly installed, 0 to remove and 361 not upgraded.`

at the same time, @extra/r-aa-fatty22-006/2:
`
[2024-08-26T21:46:55.583Z] + echo 'Build on debian'
[2024-08-26T21:46:55.583Z] Build on debian
[2024-08-26T21:46:55.583Z] ++ id -u
[2024-08-26T21:46:55.583Z] + '[' 6213 -ne 0 ']'
[2024-08-26T21:46:55.583Z] + sudo apt-get install -y debhelper
[2024-08-26T21:46:55.583Z] E: Could not get lock /var/lib/dpkg/lock - open (11: Resource temporarily unavailable)
[2024-08-26T21:46:55.583Z] E: Unable to lock the administration directory (/var/lib/dpkg/), is another process using it?
[2024-08-26T21:46:55.583Z] + apt-get install -y debhelper
[2024-08-26T21:46:55.583Z] E: Could not open lock file /var/lib/dpkg/lock - open (13: Permission denied)
[2024-08-26T21:46:55.583Z] E: Unable to lock the administration directory (/var/lib/dpkg/), are you root?`